controllers/operator/mongodbshardedcluster_controller_test.go

```diff
@@ -1793,88 +1791,6 @@ func SingleClusterShardedScalingWithOverridesTestCase(t *testing.T, tc SingleClu
 	}
 }
 
-func TestSingleClusterShardedScalingWithOverrides(t *testing.T) {
```

Comment:
b: Was this test interfering with the static container changes ?

Reply:
thanks! that shouldn't have been removed. Adding it back